services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/select_user_type.jsx

```diff
@@ -56,12 +63,19 @@ class SelectUserType extends React.Component {
               text="Select this option to join your teacher’s class and complete assigned activities."
             />
             <Card
-              header="Teacher or Guardian"
+              header="K-12 Teacher"
               imgAlt="Chalkboard"
               imgSrc={teacherChalkboardImg}
               onClick={this.handleClickTeacher}
               text="Select this option to create classes, assign activities, and view reports."
             />
+            <Card
+              header="Parent, Tutor or Caregiver"
+              imgAlt="home"
+              imgSrc={homeSchoolImg}
+              onClick={this.handleClickIndividualContributor}
+              text="Select this option to assign activities and view reports for your individual student(s)."
```

Comment:
I get that this is the spec but doesn't this make it seem like the functionality is actually different than for a K-12 teacher? Like they still have to make a class. Not saying you should do anything differently, just commenting.